### ISSUE TITLE
decoration-3.jpg mirrored left-right

### ISSUE THREAD

**Issue by sm5por** [fontforge/designwithfontforge.com#206]
Is the image in file `en-US/images/decoration-3.jpg` supposed to be mirrored left-right for some reason? The lettering in the image obviously reads "Nº 1000" (and probably "GUARANTEED BRITISH" above), only in the wrong direction. I did an image search on Google, but didn't find a likely source of the image. Maybe the book should have a page listing image sources (ok, that should probably be a separate issue if I were serious about it, but I'm not planning to contribute one anyway as I don't have the information).

**Comment by ctrlcctrlv** [comment]
Hello. What page is it on?

**Comment by frank-trampe** [comment]
The image looks like a woodcut, which is naturally inverted relative to the page that it prints.

**Comment by sm5por** [comment]
[http://designwithfontforge.com/en-US/What_Is_a_Font.html](http://designwithfontforge.com/en-US/What_Is_a_Font.html)
Woodcut, ah right, that could be the reason. Wasn't immediately obvious to me from the context though.

**Comment by sm5por** [comment]
Since the book is about typeface design, not woodcut xylography, it may be a little far-fetched, but I'm not going to complain loud about it...

**Comment by sm5por** [comment]
... so I guess I should close this issue then (if someone else thinks it requires attention, it may of course be reopened).

**Comment by frank-trampe** [comment]
I likewise think it a small problem. It's a nice image, so I don't want to remove it, and captioning it or explaining it in the text would be inconsistent with the rest of the page. Thanks for being vigilant, though. We strive for a high-quality product and appreciate feedback.

**Comment by ctrlcctrlv** [comment]
Hi, thanks for showing me where it's linked. It's not a problem to reverse the image. I'll reopen this and submit a PR later.

**Comment by frank-trampe** [comment]
@ctrlcctrlv, are you sure about reversing it? It seems like a mixed bag to me.

**Comment by ctrlcctrlv** [comment]
@frank-trampe I vote to reverse it as we're showing examples of typography, not teaching about its history. I have had several books that include photos of wood block typography that were reversed to aid readers, even historical texts. However, do you not like the idea?

**Comment by frank-trampe** [comment]
I'm not opposed, just indifferent and not voting either way. If we flip it, perhaps we ought to update the alt text or the image metadata to reflect that it's reflected so as to avoid confusing the curious?

**Comment by ctrlcctrlv** [comment]
Sure, alt text sounds good to me. I'll reverse it losslessly via the JPEG orientation API so it'll be in the metadata as well.

**Comment by skef** [comment]
I dunno ... could be a selfie.

**Comment by JoesCat** [comment]
Appears right to left (Firefox).
Downloaded image.
Gwenview displays it as left to right.
GIMP says the EXIF information is wrong (do you want to correct it?). No==same as Firefox, Yes==seen like Gwenview.

**Comment by frank-trampe** [comment]
Oh. That's odd. I can confirm that it displays in its original form in Firefox.

**Comment by JoesCat** [comment]
Firefox 60.9, 32bit, linux - maybe a more recent version is corrected.

**Comment by frank-trampe** [comment]
@JoesCat, to be clear, by "original form", I meant the same thing that you were seeing in Firefox.

**Comment by ctrlcctrlv** [comment]
It's working here. Could be caching, or outdated Firefox. Thanks for noticing I forgot the precompressed image @JoesCat 

Firefox 60 ESR is EOL as of October 2019. 

![image](https://user-images.githubusercontent.com/838783/83838323-b71e7100-a6ad-11ea-8957-e59d3db600ff.png)

**Comment by frank-trampe** [comment]
@ctrlcctrlv, my comment was comparing the state before and after #207, not the state after #208.

**Comment by ctrlcctrlv** [comment]
@frank-trampe #208 wasn't merged, #209 closed #208.

**Comment by frank-trampe** [comment]
Ah. I was off slightly, but you know what I mean.

**Comment by ctrlcctrlv** [comment]
@frank-trampe So did #209 fix it?

**Comment by frank-trampe** [comment]
No. I'm not sure what's going on. It seems highly unlikely to be a Firefox bug fixed between version 76 and version 77, but I can't think of any other explanation.

![Screenshot from 2020-06-05 11-06-25](https://user-images.githubusercontent.com/2512464/83898704-e4eece80-a71c-11ea-96bb-895d7294a8c5.png)
![Screenshot from 2020-06-05 11-06-28](https://user-images.githubusercontent.com/2512464/83898751-f33cea80-a71c-11ea-8241-2cbf6daf70c5.png)
![Screenshot from 2020-06-05 11-06-57](https://user-images.githubusercontent.com/2512464/83898759-f59f4480-a71c-11ea-9fb5-9b67bfda9d1a.png)

**Comment by JoesCat** [comment]
@frank-trampe - if you remove firefox out of the equation....
```bash
git fetch origin
git merge origin/gh-pages
gwenview en-US/images/decoration-3.jpg
gimp en-US/images/decoration-3.jpg
```
gwenview shows left to right
gimp stops first and asks what do you want to do with this? - EXIF appears wrong.
 if you still see it reflected on the website using Firefox 76/77, it might be due to distro jpg library.
I'm guessing @ctrlcctrlv is using a Windows version of Firefox with included jpg library which might be different.


@ctrlcctrlv - the uncompressed image is 4000 x 3000,
when you scale down to 960 x 720, we can expect plenty of info to be removed. I chose 80% on saving - as a photograph, it seems ok since we're working with jpg, and that is designed to be lossy if you want to keep file size lower. If it was a diagram, we definitely wouldn't want it lossy, and we really should choose something other than jpg, like gif or png.

**Comment by ctrlcctrlv** [comment]
The comment about Windows is interesting. It seems like these days browser vendors are moving away from using OS libraries as much as possible, but I booted up Xubuntu:

It seems that both @frank-trampe and I neglected to update for the same length of time.

![image](https://user-images.githubusercontent.com/838783/83923944-2e98e280-a738-11ea-98fd-1098c69c5e1b.png)

So, `# apt upgrade firefox`...

![image](https://user-images.githubusercontent.com/838783/83924021-61db7180-a738-11ea-9699-bf4dfc724284.png)

It's certainly a Firefox bug.
